src/d3d12/d3d12_renderer.cpp

```diff
@@ -1348,4 +1349,9 @@ namespace wr
 		}
 	}
 
+	void D3D12RenderSystem::CreateDefaultResources()

```

Comment:
the default textures should be moved in here probably.

Reply:
moved the default textures out of texture pool and gave it to the renderer